### ISSUE TITLE
Add option to set a save folder per website

### ISSUE THREAD

**Issue by ZeroCool940711** [fmd-project-team/FMD#246]
Hi there, it would be nice if FMD had an option so web can set where to save mangas per website instead of just a global folder or a per manga folder as it is right now, sometimes we want to download a manga from multiple websites as they might have different fan subs or some chapters that the first site didn't have, also if we want to download mangas and also hentai mangas from multiples sites we might want to save them on another folder to organize things.

**Comment by Tmp341** [comment]
I think what you want is ``Save to`` option under ``Websites->Advanced->X Site`` tab.

**Comment by ZeroCool940711** [comment]
@Tmp341 Yeah, I want an option to set the save folder as a per website basic, it should go in `Websites⇾Advanced→SiteName`, right now you can only set a global `Save To` folder on `Options->Save To` and you can also save each manga on a different folder if before downloading you set a `Save To` path but there is no way to have a default `Save To` folder per website, I would be happy if at least we had an option to set a folder for normal mangas or per language and another folder for `H-Sites`, also would be nice to have an option to save the manga info to a file, when you have too many mangas (Hentai or non-hentai) it becomes a problem to manage mangas and its not even possible to know what category a manga belongs to. I have around 1 TB of manga/hentai and it's a huge problem to manage them.

**Comment by Tmp341** [comment]
1 TB? Whoa. 😅

**Comment by ZeroCool940711** [comment]
I know, it's a lot of mangas, my goal is to download everything from TumangaOnline and other websites so I have everything on my computer in case my internet goes down but TuMangaOnline only let me download one manga at the time and its slow as hell and I also have a 1 TB monthly bandwidth limit so I have to download things slowly, I've downloaded all that in a few months and still have a lot of mangas to download but its becoming a problem to keep track of things once they are on my computer, im having Big Data Categorization problems right now xD

**Comment by SDXC** [comment]
Has been implemented. In the advanced website settings you will find a new option to override the default "save to" path per website. If nothing is set, the global default will be used. This settings does not override the paths set in the favorites.
